include/mw/models/crypto/Bech32Address.h

```diff
@@ -21,20 +21,20 @@ class Bech32Address : public Traits::ISerializable, public Traits::IJsonable
     static Bech32Address FromString(const std::string& address)
     {
         auto decoded = bech32::Decode(address);
-        if (decoded.first.empty()) {
+        if (decoded.first.empty() || decoded.second.empty()) {
             ThrowDeserialization("Failed to decode Bech32Address.");
         }
 
-        std::vector<uint8_t> addr;
-        ConvertBits<5, 8, false>([&](uint8_t c) { addr.push_back(c); }, decoded.second.begin(), decoded.second.end());
+        std::vector<uint8_t> addr = { decoded.second[0] };
+        ConvertBits<5, 8, false>([&](uint8_t c) { addr.push_back(c); }, decoded.second.begin() + 1, decoded.second.end());
```

Comment:
Hmm, why was this change needed?

Reply:
The first byte is a version byte and isn't supplied to ConvertBits according to the litecoin src.

Reply:
Rethinking this, perhaps we should make the version byte explicit so that if the format of the MWEB addresses changes in future we can increment the version number.